test/integration/ggrc/converters/test_export_with_sql_error.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (C) 2020 Google Inc.
+# Licensed under http://www.apache.org/licenses/LICENSE-2.0 <see LICENSE file>
+
+"""Tests for export with SQL error"""
+
+import json
+
+from datetime import datetime
+
+import mock
+import flask
+
+from appengine import base
+from sqlalchemy import exc
+
+from ggrc import db
+from ggrc.models import all_models
+from ggrc.views.converters import run_export
+from integration.ggrc import TestCase
+from integration.ggrc.models import factories
+
+
+@base.with_memcache
+class TestExportSQLError(TestCase):
+  """Test for failed exoprt due SQL error"""
+
+  def setUp(self):
+    super(TestExportSQLError, self).setUp()
+    self.client.get("/login")
+    self.headers = {
+        'Content-Type': 'application/json',
+        "X-Requested-By": "GGRC",
+        "X-export-view": "blocks",
+    }
+
+  @mock.patch("ggrc.notifications.common.send_email")
+  def test_operational_error(self, _):
+    """Test export with raise sqlalchemy exception"""
+    user = all_models.Person.query.first()
+    flask.g.__setattr__('_current_user', user)
+    op_type = all_models.BackgroundOperationType.query.filter_by(
+        name='export'
+    ).first()
```

Comment:
`.one()` would be better here. It is not supposed to have more then one `BackgroundOperationType` with name `"export"`. Please fix it.
```suggestion
    ).one()
```